### PR TITLE
redesign and new features for djs page

### PR DESCRIPTION
**Pull request by katiecai**
## Types of changes

- [ ] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Refactor (change which changes the codebase without affecting its external behavior)
- [ ] Non-breaking change (fix or feature that would causes existing functionality to work as expected)
- [ ] Breaking change (fix or feature that would cause existing functionality to __not__ work as expected)

## Purpose

design changes: circle images now squares, displays bio on hover
features: live search, featured current DJ for DJ hosting current show

## Approach

changed some css. added some react components. made some state changes.

## Checklist

- [x] My branch follows the branch naming scheme of UCLA Radio, and can merge into `master` without error.
- [x] My code follows the code style of this project, and I have linted it to confirm this.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] All new and existing tests passed

## Screenshots
DJs page
djs page

DJ on Hover
djs page on hover

Search feature
djs page search

Current DJ (image doesn't show up on dev)
screen shot 2019-01-14 at 5 03 29 pm

### CODE REVIEW

File: client/react/frontpage/components/DJList/DJSearchBar.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const DJSearchBar = React.createClass({
+  render() {
+    return (
+      <div className="djSearchBar">
+        <input
+          placeholder="Search for a DJ..."
+          onChange={this.props.onChange}
+        />
```
[the remark this answers is not in the document]
Comment (tanzeelak) [reply client/react/frontpage/components/DJList/DJSearchBar.js]: @katiecai would you be able to quickly address this?

[the remark this answers is not in the document]
Comment (katiecai) [reply client/react/frontpage/components/DJList/DJSearchBar.js]: done!

**Comment by tanzeelak** [comment]
@katiecai can you add screenshots of what it currently looks like?

**Comment by katiecai** [comment]
open sesame

**Pull request merged**